### DISCUSSION TITLE
Do you think there needs to be more multiplayer gamemodes in Lazer?

### DISCUSSION THREAD

**Discussion by Hissymaster** [ppy/osu#27873] [Polls]
...should be self explanantory.

**Comment by Givikap120** [comment]
IDK about gamemodes but winconditions - for sure
pp is the main one I care about but there would be good if there was big amount of customization, like manually assigning handicap/multiplier to some player / mod

**Comment by Miralko** [comment]
i think yeah, for large groups or creators/streaming pulls of groups, it could be fun to somehow increase the limit or at least let excess players spectate any/multiple plays or even play alongside without necessarily following the same start point of other players

**Reply by Hissymaster** [reply]
...There is no limit on multiplayer rooms in lazer already?

**Comment by Taneta-nemausernames** [comment]
Please bring back tag co op/team tag co op, it's easily the most fun multiplayer mode regardless of skill level

**Comment by digitalis-p** [comment]
I really miss tag and it would be great to be able to play :)
